packages/esm-form-entry-app/src/app/form-schema/form-schema.service.ts

```diff
@@ -183,16 +183,12 @@ export class FormSchemaService {
     const results = new Set<string>();
     const walkQuestions = (questions: Array<Questions>) => {
       for (const question of questions) {
-        if (typeof question.concept === 'string') {
+        if (!question.label && typeof question.concept === 'string') {
           results.add(question.concept);
         }
 
-        if (typeof question.questionOptions?.concept === 'string') {
```

Comment:
I don't *think* these concepts are actually used in labels, which is why they are skipped here.

Reply:
I was wrong about this...